### PR TITLE
Error bar computation for LMP script

### PR DESCRIPTION
**Pull request by kadupitiya**
@jadhao: Please check and approve.

### CODE REVIEW

File: bin/infiles/in.lammps.chargedsurface.template [hunk 1]
```diff
@@ -26,6 +26,8 @@ variable totalSteps equal ${stepsBeforeEq}+${stepsAfterEq}
 variable densityDataFrq equal USERINPUT_FRQ
 variable densityDataSamples equal ${stepsAfterEq}/${densityDataFrq}
 variable binWidth equal 0.05 #0.05023206751
+variable densityWriteFrq equal ${totalSteps}*0.1
```
Comment (jadhao) [inline bin/infiles/in.lammps.chargedsurface.template:29]: @kadupitiya do you want stepsAfterEq here or totalSteps is fine?

**Comment by jadhao** [comment]
@kadupitiya made 1 comment on the LMP input script. python fn to compute error bars looks good but did not check closely.

File: bin/infiles/in.lammps.chargedsurface.template (hunk 1 above)
Comment (kadupitiya) [reply bin/infiles/in.lammps.chargedsurface.template:29]: @jadhao: It should be ${totalSteps}*0.1:
Ex: totalSteps=1000 and stepsAfterEq=900.
So when I use ave/chunk ${densityDataFrq} ${densityDataSamplesErrorbars} ${densityWriteFrq} command, it should dump a new density file every 100 steps not every 90.

Comment (jadhao) [reply bin/infiles/in.lammps.chargedsurface.template:29]: @kadupitiya sounds good. approving

**Pull request merged**